biomejs/biome
# docs: rename rome to biome

## Summary


This commit changes instances of rome that were left in to the new name of biome



## Test Plan


1. Check that the website loads and content, styling, and functionality is correct
2. Check that the playground is functioning as normal
